### PR TITLE
Updated default values

### PR DESCRIPTION
**Pull request by eldadfux**
## What does this PR do?

Made sure all response objects have default values.

## Test Plan

Use existing tests

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/Appwrite/Utopia/Response/Model/Execution.php [hunk 1]
```diff
@@ -13,46 +13,55 @@ public function __construct()
...
             ->addRule('stderr', [
                 'type' => self::TYPE_STRING,
                 'description' => 'The script stderr output string.',
+                'default' => '',
                 'example' => '',
             ])
             ->addRule('time', [
                 'type' => self::TYPE_FLOAT,
                 'description' => 'The script execution time in seconds.',
+                'default' => '',
```
Comment (TorstenDittmann) [inline src/Appwrite/Utopia/Response/Model/Execution.php:64]: I assume '' isn't a valid float value 🤔

File: src/Appwrite/Utopia/Response/Model/File.php [hunk 2]
```diff
@@ -13,37 +13,44 @@ public function __construct()
             ->addRule('$id', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File ID.',
+                'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
             ->addRule('$permissions', [
                 'type' => Response::MODEL_PERMISSIONS,
                 'description' => 'File permissions.',
+                'default' => '',
```
Comment (TorstenDittmann) [inline src/Appwrite/Utopia/Response/Model/File.php:22]: Can this also be `new \stdClass` on default?

File: src/Appwrite/Utopia/Response/Model/File.php [hunk 3]
```diff
@@ -13,37 +13,44 @@ public function __construct()
...
             ->addRule('mimeType', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File mime type.',
+                'default' => '',
                 'example' => 'image/png',
             ])
             ->addRule('sizeOriginal', [
                 'type' => self::TYPE_INTEGER,
                 'description' => 'File original size in bytes.',
+                'default' => '',
```
Comment (TorstenDittmann) [inline src/Appwrite/Utopia/Response/Model/File.php:53]: Must be `0` 🙂

File: src/Appwrite/Utopia/Response/Model/Execution.php (hunk 1 above)
Comment (eldadfux) [reply src/Appwrite/Utopia/Response/Model/Execution.php]: Fixed :)

File: src/Appwrite/Utopia/Response/Model/File.php (hunk 2 above)
Comment (eldadfux) [reply src/Appwrite/Utopia/Response/Model/File.php]: Fixed :)

File: src/Appwrite/Utopia/Response/Model/File.php (hunk 3 above)
Comment (eldadfux) [reply src/Appwrite/Utopia/Response/Model/File.php]: Fixed...

Hehe you got to love a good code review :)

File: src/Appwrite/Utopia/Response/Model/File.php [hunk 4]
```diff
@@ -13,37 +13,44 @@ public function __construct()
...
             ->addRule('name', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File name.',
+                'default' => '',
                 'example' => 'Pink.png',
             ])
             ->addRule('dateCreated', [
                 'type' => self::TYPE_INTEGER,
                 'description' => 'File creation date in Unix timestamp.',
+                'default' => '',
```
Comment (TorstenDittmann) [inline src/Appwrite/Utopia/Response/Model/File.php:35]: Found another, needs to be `0` 😄

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

File: src/Appwrite/Utopia/Response/Model/File.php (hunk 4 above)
Comment (eldadfux) [reply src/Appwrite/Utopia/Response/Model/File.php]: Fixed :)

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**